example/lib/main.dart

```diff
@@ -15,9 +15,11 @@ class MyApp extends StatefulWidget {
 
 class _MyAppState extends State<MyApp> {
   String _platformVersion = 'Unknown';
+  FlutterSplit flutterSplit = FlutterSplit();
 
   @override
   void initState() {
+    flutterSplit.initializeSdk("18293178372");
     super.initState();
     initPlatformState();
```

Comment:
This seems to be a remnant from the default plugin example.
Must remove if not relevant to split.

Reply:
Should delete the call to and definition of `initPlatformState `